presto-docs/src/main/sphinx/functions/tdigest.rst

```diff
@@ -0,0 +1,76 @@
+==================
+T-Digest Functions
+==================
+
+Presto implements two algorithms for estimating rank-based metrics, `quantile
+digest <http://dx.doi.org/10.1145/347090.347195>`_ and `T-digest 
+<https://doi.org/10.1016/j.simpa.2020.100049>`_.  T-digest has `better
+performance <https://arxiv.org/abs/1902.04023>`_ in general while the Presto
+implementation of quantile digests supports more numeric types.
```

Comment:
In these headers, I would also mention:

*) T-Digest has better accuracy at the tails, often dramatically better, but may have worse accuracy at the median, depending on the compression factor used.  In comparison, Q-Digest supports a maximum rank error, which guarantees relative uniformity of precision along the quantiles.
*) T-Digest is empirically proven to support lossless merge, although this is not formally proven.